src/view/components/namespaceComponent.jsx

```diff
@@ -1,61 +1,61 @@
-import Select from "@react/react-spectrum/Select";
 import React from "react";
-import Textfield from "@react/react-spectrum/Textfield";
 import PropTypes from "prop-types";
-import WrappedField from "./wrappedField";
+import { Item } from "@adobe/react-spectrum";
+import { Picker, TextField } from "./formikReactSpectrum3";
 
-const getSelectedNamespace = (options, namespace) => {
-  if (options.length < 1) {
+const getSelectedNamespace = (namespaces, selectedNamespaceCode) => {
```

Comment:
We have several references to `namespace`. Sometimes it referred to the namespace object, while sometimes it referred specifically to the namespace code that's on a namespace object. I did some renaming to disambiguate.